tests/x11/libreoffice/libreoffice_mainmenu_components.pm

```diff
@@ -127,6 +127,20 @@ sub run {
     type_string "calc";                                                             #open calc
     assert_and_click 'overview-office-calc';
     assert_screen 'test-oocalc-1';
+    if (!match_has_tag('ooffice-tip-of-the-day')) {
+        # Sometimes the dialog appears after a short delay
```

Comment:
I don't understand why that should be needed. `assert_screen` by default has a timeout of 30s meaning in line L129 we already wait for 30s anyway. If this is not enough, simply add a timeout value there?

Reply:
'test-oocalc-1' are needles with and without the tip dialog. First the program is shown, then the dialog appears - but in approximately 19/20 cases, this is the same frame.